src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -208,6 +209,29 @@ internal CookieContainer CookieContainer
             }
         }
 
+        internal int MaxAutomaticRedirections
+        {
+            get
+            {
+                return _maxAutomaticRedirections;
+            }
+
```

Comment:
We should be consistent with gaps between get and set, either never having them or always having them.  Even in this one commit some of them have a gap and some don't.